engine/gamesys/src/gamesys/gamesys_resource.cpp

```diff
@@ -266,19 +265,4 @@ namespace dmGameSystem
 
         return dmResource::SetResource(factory, params.m_PathHash, (void*) &recreate_params);
     }
-
-    dmResource::Result ReleaseDynamicResource(dmResource::HFactory factory, dmGameObject::HCollection collection, dmhash_t path_hash)
-    {
-        HResourceDescriptor rd = dmResource::FindByHash(factory, path_hash);
-        if (!rd)
-        {
-            return dmResource::RESULT_RESOURCE_NOT_FOUND;
-        }
-
-        // This will remove the entry in the collections list of dynamically allocated resource (if it exists),
-        // but we do the actual release here since we allow releasing arbitrary resources now
-        dmGameObject::RemoveDynamicResourceHash(collection, path_hash);
-        dmResource::Release(factory, dmResource::GetResource(rd));
```

Comment:
I don't use `RemoveDynamicResourceHash` in gui anymore. 
Also, in all the other places we use `RemoveDynamicResourceHash` and `AddDynamicResourceHash` together **with** `gamesys_resource.cpp` APIs, not inside it